src/components/EnvironmentOverride/ConfigMapOverrides.tsx

```diff
@@ -62,11 +62,11 @@ export default function ConfigMapOverrides({ parentState, setParentState }: Conf
                     id: configmapRes.result.id,
                     configData: configmapRes.result.configData || [],
                 })
-                if (!appChartRefRes.result) {
+                if (!isJobView && !appChartRefRes.result) {
```

Comment:
refactor this code so that incase of isJobView we don't hit the appChartRepo API

Reply:
done